content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Application server configuration
+linkTitle: Server config
```

Comment:
```suggestion
linkTitle: Application server
```

For me "application server" is something different to just "server", for a Java developer this might be less confusing this way. I would even be OK with `Application server config` (or `App server config`)

Reply:
Remember that the linkTitle is only used in the left nav. When someone visits the page, they see the full title.

That being said, if you still feel that `Server config` isn't an appropriate title, among the suggestions you make, I'd vote for `App server config`.

Reply:
Then let's go for `App server config` 👍